### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Debas-31**
- [x] Added some new tables 
- [x] Added foreign key columns to existing animals table 
- [x] used JOIN to query the data.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -49,4 +49,47 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
...
+SELECT name FROM owners O
+    JOIN animals A
+    ON O.id = A.owner_id
+    WHERE O.full_name = 'Dean Winchester' AND escape_attempts = 0;
+
+/* Who owns the most animals? */
+SELECT full_name, COUNT(name) FROM owners O
+    LEFT JOIN animals A
+    ON O.id = A.owner_id
+    GROUP BY full_name;
```
Comment (DammyShittu) [inline queries.sql:95]: - Great queries so far. 👍 However, the last query says; `Who owns the most animals?`.
After running your query, this 👇 is what I got;



To get the person who owns the most animals, I believe the query should be written this 👇 way;

```
SELECT full_name, COUNT(name) FROM owners O
LEFT JOIN animals A
ON O.id = A.owner_id   GROUP BY full_name 
ORDER BY COUNT(name) 
DESC LIMIT 1;
```
😉

File: data.sql [hunk 2]
```diff
@@ -11,4 +11,45 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) V
...
+INSERT INTO species (name) VALUES('Pokemon');
+INSERT INTO species (name) VALUES('Digimon');
+
+UPDATE animals 
+SET species_id = 1
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 2
+WHERE name NOT LIKE '%mon';
```
Comment (DammyShittu) [inline data.sql:34]: - Kindly note that animals with names ending with `mon` are expected to belong to the `Digimon` species. According to the arrangement on lines 25 and 26, kindly alter the `species_id` on lines 29 and 33 to the right `id`. 😁

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Debas-31, 👋 

You have done an amazing job so far. 

Well done 👍 

There are little issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

_Check the comments under the review._


# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @Debas-31 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**